conformance/src/main/java/build/buf/validate/conformance/cases/AnEnum.java

```diff
@@ -0,0 +1,107 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Please configure a .gitattributes file so that these are flagged as generated in follow ups: https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github